src/openassetio-python/package/openassetio/errors.py

```diff
@@ -20,9 +20,10 @@
 
 from . import _openassetio  # pylint: disable=no-name-in-module
 
-
 OpenAssetIOException = _openassetio.errors.OpenAssetIOException
 InputValidationException = _openassetio.errors.InputValidationException
 ConfigurationException = _openassetio.errors.ConfigurationException
 NotImplementedException = _openassetio.errors.NotImplementedException
 UnhandledException = _openassetio.errors.UnhandledException
+BatchElementError = _openassetio.errors.BatchElementError
```

Comment:
[fix] Add to list in `cmodule/test_errors.py`

Reply:
https://github.com/OpenAssetIO/OpenAssetIO/pull/1109/commits/e65bc0d7f63d8d4dc044a8ba031135dfe211c9a1